news/_posts/core-weekly/2020-06-02-coreweekly-22-2020.md

```diff
@@ -0,0 +1,248 @@
+* [#6](https://github.com/PrestaShop/statspersonalinfos/pull/6): Release 2.0.3, by [@Progi1984](https://github.com/Progi1984)
+* [#5](https://github.com/PrestaShop/statspersonalinfos/pull/5): Fix age data with birthday 0000-00-00 and empty currency label, by [@Matt75](https://github.com/Matt75)
+
+
+### Gamification module
+* [#73](https://github.com/PrestaShop/gamification/pull/73): Rename function name to avoid conflicts, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+
+
+### Traces
+* [#9](https://github.com/PrestaShop/traces/pull/9): Changed arguments for username & password in options, by [@Progi1984](https://github.com/Progi1984)
+
+
+### example_module_mailtheme
+* [#6](https://github.com/PrestaShop/example_module_mailtheme/pull/6): Bump jquery from 3.4.1 to 3.5.0. Thank you [@dependabot[bot]](https://github.com/apps/dependabot)
+* [#5](https://github.com/PrestaShop/example_module_mailtheme/pull/5): Bump acorn from 6.1.1 to 6.4.1. Thank you [@dependabot[bot]](https://github.com/apps/dependabot)
+* [#4](https://github.com/PrestaShop/example_module_mailtheme/pull/4): Bump lodash from 4.17.11 to 4.17.15. Thank you [@dependabot[bot]](https://github.com/apps/dependabot)
+
+
+### ps_currencyselector
+* [#17](https://github.com/PrestaShop/ps_currencyselector/pull/17): Add condition to avoid duplicate currency name, by [@NeOMakinG](https://github.com/NeOMakinG)
+
+
+### Contact informations module
+* [#32](https://github.com/PrestaShop/ps_contactinfo/pull/32): Fixed uninitialized string offset bug. Thank you [@joaolago1113](https://github.com/joaolago1113)
+
+
+### ps_buybuttonlite
+* [#32](https://github.com/PrestaShop/ps_buybuttonlite/pull/32): Bump lodash.defaultsdeep from 4.6.0 to 4.6.1 in /app. Thank you [@dependabot[bot]](https://github.com/apps/dependabot)
+* [#31](https://github.com/PrestaShop/ps_buybuttonlite/pull/31): Bump lodash.mergewith from 4.6.1 to 4.6.2 in /app. Thank you [@dependabot[bot]](https://github.com/apps/dependabot)
+* [#30](https://github.com/PrestaShop/ps_buybuttonlite/pull/30): Bump lodash from 4.17.10 to 4.17.15 in /app. Thank you [@dependabot[bot]](https://github.com/apps/dependabot)
+* [#29](https://github.com/PrestaShop/ps_buybuttonlite/pull/29): Bump fstream from 1.0.11 to 1.0.12 in /app. Thank you [@dependabot[bot]](https://github.com/apps/dependabot)
+* [#28](https://github.com/PrestaShop/ps_buybuttonlite/pull/28): Bump mixin-deep from 1.3.1 to 1.3.2 in /app. Thank you [@dependabot[bot]](https://github.com/apps/dependabot)
+
+
+### Product Comments module
+* [#42](https://github.com/PrestaShop/productcomments/pull/42): Rebuild ratings loading on product list. Thank you [@Oksydan](https://github.com/Oksydan)
+
+
+### Email subscription module
+* [#35](https://github.com/PrestaShop/ps_emailsubscription/pull/35): Fixed the email when user is registering (to the site and the newsletter simultaneously). Thank you [@Codencode](https://github.com/Codencode)
+
+
+### vatnumber
```

Comment:
No "pretty" name

Did you read the PR content before opening it ? 🤔

Reply:
```suggestion
### European VAT number
```